editor.planx.uk/src/pages/FlowEditor/lib/store/team.ts

```diff
@@ -65,6 +66,12 @@ export const teamStore: StateCreator<
     theme: get().teamTheme,
   }),
 
+  create: async (newTeam) => {
```

Comment:
We should call this `createTeam()` 👍 

Even though `create` is a useful and clear name in the context of the `TeamStore`, we actually merge all stores into one (checkout `editor.planx.uk/src/pages/FlowEditor/lib/store/index.ts`). 

In isolation in another component `useStore.getState().create()` doesn't have much context.